### PR TITLE
Gymnastic-strength-ON-321

### PR DESCRIPTION
**Pull request by AviralSingh111**

### CODE REVIEW

File: src/features/Workout/SectionDetail.js [hunk 1]
```diff
@@ -346,21 +348,51 @@ const SectionDetail = () => {
...
-                      label={input.label}
-                    />
-                  ))}
-                </div>
+
+               <div>
+        <h2 className="workout-gradient-text my-4 text-2xl">
+          Data Inputs
+        </h2>
+        {dataInput.slice(0, 2).map((input, index) => (
```
Comment (SaurabhSahu01) [inline src/features/Workout/SectionDetail.js]: why is slice being used here?

File: src/features/Workout/SectionDetail.js [hunk 2]
```diff
@@ -346,21 +348,51 @@ const SectionDetail = () => {
...
+            key={index}
+            inputId={input.id}
+            inputType={input.type}
+            inputOptions={input.options}
+            placeholder={input.label}
+            label={input.label}
+          />
+        ))}
+        
+        {name === "Gymnastic Strength" && dataInput.length > 2 && (
```
Comment (SaurabhSahu01) [inline src/features/Workout/SectionDetail.js]: can we set the condition based on code of the workout section?

File: src/features/Workout/SectionDetail.js (hunk 1 above)
Comment (AviralSingh111) [reply src/features/Workout/SectionDetail.js]: Slice is used so the first two data inputs are in a separate section and the third one is under Max effort section.

File: src/features/Workout/SectionDetail.js (hunk 2 above)
Comment (AviralSingh111) [reply src/features/Workout/SectionDetail.js]: Good Point, I can use the code instead of the name.

**Review by SaurabhSahu01 — APPROVED** [review]

**Pull request merged**